ssl/ssl_test.go

```diff
@@ -315,6 +398,27 @@ func TestCreateCA(t *testing.T) {
 
 		assert.NotEqual(t, secrets.Data[certInfo[certID].PrivateKeyName], []byte{})
 		assert.NotEqual(t, secrets.Data[certInfo[certID].CertificateName], []byte{})
+		assert.Regexp(t, "(?s)\\A-----BEGIN CERTIFICATE-----\\n.*\\n-----END CERTIFICATE-----\\n\\z",
```

Comment:
Use backticks for the string delimiter to skip the need to escape backslashes.  Too bad there isn't `assert.HasPrefix` or something like that…

Reply:
Fixed in 8cf5338